student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
```

Comment:
- [OPTIONAL] As the `name` and `parent_permission` parameters have already the default value defined in the initialize you don't need to give them again a default value when you call them in `super()` 👉🏾 `super(age, name, parent_permission)` 👍🏾